### PR TITLE
docs(home): add content to home page

### PR DESCRIPTION
**Pull request by yashints**
added content to main page: installation guide #17

### CODE REVIEW

**Comment by yashints** [comment]
@JWO719 I am following the guidelines (header / body) with a line break, should I just enter the header? which white space is causing the failure?

**Comment by JWO719** [comment]
the commit message should be in format: <type>(<scope>): <subject>
And I think main page and installation causes the failing travis build. Maybe replace it with 'instructions" or something like that.

**Comment by yashints** [comment]
thanks for the tips @JWO719

File: src/app/rxjs/components/installation/installation.component.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-installation',
+  templateUrl: './installation.component.html',
+  styleUrls: ['./installation.component.scss']
+})
+export class InstallationComponent implements OnInit {
+  importPatching = `import { Observable} from 'rxjs/Observable';
+import 'rxjs/add/observable/of';
```
Comment (ashwin-sureshkumar) [inline src/app/rxjs/components/installation/installation.component.ts]: Looks like this is misplaced code. Can you please remove these ?

File: src/app/rxjs/rxjs.component.html [hunk 2]
```diff
@@ -1,3 +1,44 @@
...
+    </mat-card>
+
+
+    <div class="tiles">
+        <div class="tile">
+            <h3>The ReactiveX library for JavaScript</h3>
+
+            <p>
+                RxJS is a library for reactive programming using Observables, to make it easier to compose asynchronous or callback-based
+                code. This project is a rewrite of Reactive-Extensions/RxJS with better performance, better modularity, better
```
Comment (ashwin-sureshkumar) [inline src/app/rxjs/rxjs.component.html]: Can you please utilize translation for this content ?

File: src/app/rxjs/components/installation/installation.component.ts (hunk 1 above)
Comment (yashints) [reply src/app/rxjs/components/installation/installation.component.ts]: @ashwin-sureshkumar  These are not misplaced codes, they are part of the installation process, it was making the angular compiler complaining about having not recognisable code although it was string in template.
That's why I put them in a variable in the component. I can try to put them in the html template if that's not aligned with the guidelines.

**Comment by ashwin-sureshkumar** [comment]
@yashints My bad. Yeah if you could move this into the template would be great

File: src/app/rxjs/rxjs.component.html (hunk 2 above)
Comment (yashints) [reply src/app/rxjs/rxjs.component.html]: sure

File: src/app/rxjs/components/installation/installation.component.ts (hunk 1 above)
Comment (yashints) [reply src/app/rxjs/components/installation/installation.component.ts]: Don't worry about it I moved them to html, update coming soon

**Comment by yashints** [comment]
@ashwin-sureshkumar done

**Comment by ashwin-sureshkumar** [comment]
@yashints - nice work !! Thanks for the PR !!! :)

**Comment by ashwin-sureshkumar** [comment]
@JWO719 can you review and approve please

**Comment by JWO719** [comment]
Yeah of course. Sorry for being that late. I will have some time for a review tomorrow and give some feedback

**Comment by JWO719** [comment]
WOOP WOOP @yashints PR merged. Well Done!

**Comment by yashints** [comment]
Thanks @JWO719 and @ashwin-sureshkumar and especially @ladyleet

**Pull request merged**